### DISCUSSION TITLE
Password not saved

### DISCUSSION THREAD

**Discussion by gfreccia** [ltsp/ltsp#363] [Q&A]
Hi All,

I'm struggling with password not saved on network sharing.
If I open a Samba shared folder from an user on the LTSP server and set "Remember Password", it works properly and no need to type the password anymore.
But when I open the same user on a FAT Client the password is not saved and I've to retype every time after logged in, even if I set again "Remember".
A more simple option colud be to set the auto-mount on lts.conf, but, how?

Thank you!

**Comment by alkisg** [comment]
1. Are you using autologin? Mention both the server and the client.
2. Run `seahorse` and check if your login keyring is unlocked on login, both on the server and the client.

**Comment by gfreccia** [comment]
Hi Alkis,

thanks for the prompt answer!
No autologin and I've unlocked the keyring, but no luck!
In any case I think that a more elegant solution is to mount the shared folder at boot.
I know that it's possible to set on lts.conf.
Does it work also for SMB share or for NFS share only?
What's the command to set?

**Comment by alkisg** [comment]
To mount it using /etc/fstab, google for "how to mount samba /etc/fstab".
Once you find the correct fstab line, you can put it in FSTAB_x="...." in ltsp.conf.

**Comment by gfreccia** [comment]
Hi Alkis,

this is the command, and it works:

_FSTAB_1="//172.50.7.240/UFFICIO  /mnt  cifs  username=ufficio,password=xxxxxxx,iocharset=utf8  0  0"_

But, how I can set to mount on a folder (e.g. OFFICE) inside Document folder of an user?

**Reply by enaut** [reply]
you probably still mount it elsewhere and then symlink with `ln -s /mnt /home/userpeter/OFFICE` But every user will be able to access the files under `/mnt/`. The other option is to write a script that does what you want in bash or python.

**Comment by gfreccia** [comment]
Hi!

But it is not possible to set it based on User on the lts.conf?
I see that it's possible to set command for single client, per MAC, IP..., it is not possible to set it per User?

**Comment by alkisg** [comment]
FSTAB_x means "put something in /etc/fstab". This means "for all users".
FSTAB_PER_USER entries can't be supported, as Linux doesn't have /etc/user/fstab.

The Linux way of running programs and mount commands after login is to place them in /etc/xdg/autostart, or to create systemd user units, or to create/override login sessions like "method 2" in https://github.com/ltsp/ltsp/discussions/361#discussioncomment-254768.
These aren't specific to LTSP so they're not documented upstream in LTSP. But if someone wants to document them, he can use the LTSP wiki for that.

**Comment by gfreccia** [comment]
Hi Alkis,

clear!
So, the most simple solution is to mount with "Connect to server" for each user and create a Bookmark.
I usually use this approch, but on this latest installation (Mate 20.04) I'm not able to have "save password" working!

**Comment by gfreccia** [comment]
Hi All,

still struggling!
Why if I login on the server, opening the bookmark to my shared folder it doesn't ask for password, and when I log from the client it asks always for password, also if I set "remenber password forever"?
![Schermata del 2021-01-04 18-32-10](https://user-images.githubusercontent.com/35954627/103562692-c4a60e80-4ebb-11eb-9383-b2e2c3e912f3.png)

Thanks for helping!
